xdis/opcodes/opcode_36.py

```diff
@@ -0,0 +1,88 @@
+# (C) Copyright 2016 by Rocky Bernstein
+"""
+CPython 3.6 bytecode opcodes
+
+used in scanner (bytecode disassembly) and parser (Python grammar)
+
+This is a superset of Python 3.6's opcode.py with some opcodes that simplify
+parsing and semantic interpretation.
+"""
+
+from copy import deepcopy
+
+from xdis.wordcode import _findlabels as findlabels
+from xdis.wordcode import _findlinestarts as findlinestarts
+
+import xdis.opcodes.opcode_3x as opcode_3x
+from xdis.opcodes.opcode_3x import fields2copy, hasfree, rm_op
+
+# FIXME: can we DRY this even more?
+
+opmap = {}
+opname = [''] * 256
+hasconst = []
+hasjrel = []
+hasjabs = []
+
+def def_op(name, op):
+    opname[op] = name
+    opmap[name] = op
+
+for object in fields2copy:
+    globals()[object] =  deepcopy(getattr(opcode_3x, object))
+
+# Below are opcodes changes since Python 3.2
+
+rm_op('STOP_CODE', 0, locals())
+rm_op('STORE_LOCALS', 69, locals())
+
+# These are new since Python 3.3
+def_op('YIELD_FROM', 72)
+def_op('LOAD_CLASSDEREF', 148)
+
+# These are removed since Python 3.4
+rm_op('WITH_CLEANUP', 81, locals())
+
+# These are new since Python 3.4
```

Comment:
This is a typo on my part. It should have been 3.5. And  I need to fix in the corresponding 3.5 file.